modules/ROOT/pages/to-request-access.adoc

```diff
@@ -69,7 +69,7 @@ The OAuth 2 grant type:
 
 == API Groups
 
-When you request access to an API group, Exchange automatically compares all available contracts and applies the least restrictive contract. The application uses that contract to access the API group instance.
+When you request access to an API group, the least restrictive of the available contracts is applied. The application uses that contract to access the API group instance.
```

Comment:
A similar suggestion as before to simplify the wording: "...the least restrictive available contract is applied."

Reply:
Agreed and fixed!